src-tauri/src/node.rs

```diff
@@ -258,3 +260,19 @@ fn database_config(base_path: &Path, cache_size: usize, role: &Role) -> Database
         cache_size,
     }
 }
+
+pub async fn node_controller(path: String, node_name: String, command: &str) {
+    static mut NODE_HANDLE: Option<JoinHandle<()>> = None;
+    match command {
+        "start" => unsafe {
+            NODE_HANDLE = Some(init_node(path.into(), node_name).await.unwrap());
+        },
+        "restart" => unsafe {
```

Comment:
Why not `stop`? So you could call `Stop` and then `Start` to get effective restart.

Reply:
:ok_hand: